### PR TITLE
Make Saltern driveable

### PR DESCRIPTION
**Pull request by metalgearsloth**
There's still loootttsssss of work that needs doing but all the relevant dynamictrees are now relative so we can at least drive saltern around at more than 5fps. Some of the hard work is out of the way too at least.

Requires https://github.com/space-wizards/RobustToolbox/pull/1848

:cl:
- add: Add Station driving

Mapping changes:
The grid has fixtures stored on it (mainly for the sake of fixing tests) + I mapped in a console to command.

### CODE REVIEW

File: Content.Server/Shuttles/HelmsmanSystem.cs [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+{
+    internal sealed class HelmsmanSystem : SharedHelmsmanSystem
+    {
+        public override void Initialize()
+        {
+            base.Initialize();
+            SubscribeLocalEvent<HelmsmanConsoleComponent, ComponentShutdown>(HandleHelmsmanShutdown);
+            SubscribeLocalEvent<PilotComponent, ComponentShutdown>(HandlePilotShutdown);
+            SubscribeLocalEvent<HelmsmanConsoleComponent, ActivateInWorldEvent>(HandleHelmsmanInteract);
+            SubscribeLocalEvent<PilotComponent, MoveEvent>(HandlePilotMove);
```
Comment (mirrorcult) [inline Content.Server/Shuttles/HelmsmanSystem.cs:20]: i dont like that this is called every time any human moves (assuming they would all have a pilotcomponent), you could maybe add a comp to the entity that's currently piloting a helmsman console and then subscribe directed to that instead? (and remove the comp when they move away, etc)

Comment (mirrorcult) [reply Content.Server/Shuttles/HelmsmanSystem.cs:20]: if you do that probably just move all the PilotComponent data onto that and then use a different metric for whether an entity can pilot the shuttle (caninteract?)

Comment (metalgearsloth) [reply Content.Server/Shuttles/HelmsmanSystem.cs]: Problem is that RemoveComponent is currently bugged and doesn't sync the removal from server to client so I hadn't coded around this yet.

**Comment by metalgearsloth** [comment]
Should be good for a review BUT it's currently a DNM due to the RemoveComponent<T> bug.

**Pull request merged**